content/en/docs/_index.md

```diff
@@ -0,0 +1,6 @@
+# CI Docs
```

Comment:
@stevekuznetsov @petr-muller I honestly don't know what to put here without just repeating the table of contents or actual content. Any suggestions?

Reply:
The only thing that comes to mind is a curated subset of the most important doc links, maybe linked to a persona?

If you are new to OpenShift CI, read 
- A 
- B

If you want to understand how OpenShift CI works:
- C
- D

But I quickly ran out of personas :)

Reply:
I've added a bit to that affect but I am not really convinced this adds any value

Reply:
I guess that speaks more about how (badly) our docs are structured than the fact that we do not need a landing page :( Unfortunately we seem to produce a lot of "how X works"-kind of docs that (we wish) serve both as architecture documents but also at the same time instruct the users how to achieve stuff...

I tried looking at https://kubernetes.io/docs/home/ which is structured around basic Calls to Action, but I fail to recognize similar patterns in our docs. Could we at least plagiarize their visual with the prominent sections? I think that nobody will ever read the actual full sentences on the landing page.

We definitely seem to have some docs that are generally important and useful (and these should be linked from the landing page) and some are more niche. I'd say that the following docs should be *very* prominent on the landing page, maybe even with task-oriented titles instead of the ones that we gave to the docs themselves:

- Onboarding a New Component for Testing and Merge Automation (= "Setup CI for New Repository")
- Contributing CI Configuration to the openshift/release Repository (= "File a Pull Request to openshift/release")
- CI Operator (= "Learn More About CI Configuration")
- Multi-Stage Tests and the Test Step Registry (= "Learn More About Tests Against OpenShift Clusters")

Or something similar.

Reply:
I don't think this is really comparable, because the kube docs do not have the table of contents expanded, so those add new information there but not in our case. Also, they do not have alternate titles there, they just reference the existing titles. If we think the current naming is bad, we should change it rather than having the same page under a different title in the overview page IMHO.

The docs pages I can find seem to just have a full TOC in their overview page but do not show the expanded TOC on the side, which we do, i.E.
* Openshift: https://docs.openshift.com/container-platform/4.7/welcome/index.html
* AWS: https://docs.aws.amazon.com/

Reply:
Sorry, I accidentally had my review left as a draft.

Reply:
We certainly have vague groupings of documents. Making those stronger and making the entrypoint page useful for navigation is the point here. Listing the ToC is not. We should strive to have a landing page that overviews the types of use-cases/end-goals we have docs around, and points people to the right place. Give top-level subjects so someone does not search through the whole list. Potentially restructure our (poor) sections into more, tiered sections.

Reply:
> This is what the site looks like on my default render. The sidebar is pretty dense and hard to navigate.

You don't have a widescreen? It fits very well on one.

> We certainly have vague groupings of documents. Making those stronger and making the entrypoint page useful for navigation is the point here. Listing the ToC is not. We should strive to have a landing page that overviews the types of use-cases/end-goals we have docs around, and points people to the right place. 

Sorry, but that is not enough information to act on. Can you please be specific about which groupings you are seeing?

Reply:
I listed groups in my above comment ... if this is too vague as-is let's merge this PR (it's useful) and put the card back into the backlog.

Reply:
Group ideas were: https://github.com/openshift/ci-docs/pull/114#discussion_r598869045

Reply:
> so you're looking to onboard a repo...

There is a page called `onboarding-a-new-component`. What else would you put into that group?

> so you're looking to write a job...

CI operator doc and almost every single `How To` page, depending on what exactly you want to do

> so you're looking to interact with production clusters/registries/jobs/etc...

I don't understand what these have in common other than the word `interacting`.

Reply:
Again as noted above a number of times, just the fact that a doc is currently in the sidebar does not make it discoverable. For onboarding, I think the following docs are clearly related:

- Onboarding a New Component for Testing and Merge Automation
- Testing Operators Built With The Operator SDK and Deployed Through OLM
- Contributing CI Configuration to the openshift/release Repository

For writing new jobs:

- Add a Job to TestGrid
- Adding and Changing Step Registry Content
- Migrating CI Jobs from Templates to Multi-stage Tests
- Set Up Slack Alerts for Periodic Job Results
- Using External Images in CI

For working with downstream/ART things:

- Private Repositories and Fixing Embargoed CVEs
- OCP Builder Images

For real architecture/reference:

- Multi-Stage Tests and the Test Step Registry
- Handling Job Timeouts and Interruptions
- CI Operator

For branching and release:

- Centralized Release Branching and Config Management
- Extending OpenShift Release Gates

Think about user stories. Each of these can come after "as a user, I'm trying to ...". That's how they're connected. Obviously their themes are interrelated. Yes, this is nebulous. No, there is no perfect way to do it. Yes, changing the sidebar/directories might even be a good idea. The point of the Jira card is just to attempt to make a better landing page.

Reply:
done, ptal